### DISCUSSION TITLE
hat schon jemand HVL mit HAP-HomeMatic am Laufen?

### DISCUSSION THREAD

**Discussion by HeikoSt** [thkl/hap-homematic#487] [General]
Danke vorab für diese tolle Lösung die HM-Welt in die Home-Welt zu integrieren. Meine in HVL angelegten Geräte lassen sich über die Homematic steuern. Binde ich diese über Hap-Homematic ein und versuche sie über Apple Home zu steuern, reagieren diese nicht. Habe sowohl Harmony als auch Teufel-Raumfeld Geräte installiert, bei beiden reagiert der "Schalter" nur ganz kurz und springt dann wieder auf AUS. Geräte werden nicht geschaltet.

**Comment by robster85** [comment]
Push: Wäre sehr interessant zu erfahren, ob es da schon Lösungen gibt. Bisher werden meine Tradfri (Ikea) Geräte nicht angezeigt.

**Comment by chkania** [comment]
+1

**Comment by snookerap** [comment]
HVL ist doch nicht mehr up-to-date?!

**Comment by ChrWi** [comment]
Was ist HVL? Meine Ikea Bridge (mit Tradfri, Symfonisk) habe ich direkt über Apple HomeKit eingebunden.

**Comment by HeikoSt** [comment]
Aha

—  Von unterwegs gesendet —
—  z.Z. aber nur von zu Hause —
